package org.example.sii_task.errorHandling;

public class AlreadyExistsException {
}
